Database Dump and Load (#41)
eProsima/Fast-DDS-statistics-backend: docs/rst/types/json_tags.rst

.. include:: ../exports/alias.include

.. _types_tags:

JSON Tags
=========

> The |StatisticsBackend-api| use JSON format to retrieve information in many methods as |get_info-api|,

Reviewer: ```suggestion
The |StatisticsBackend-api| uses JSON format to retrieve information in many methods as |get_info-api|,
```

|get_graph-api| or |dump_database-api|.

.. todo::

>     In order to access every item in every JSON generated, the following tags are provided:
>     Create table

Reviewer: ```suggestion
    In order to access each item within the JSON generated file, the following tags are provided:

    Pending table creation.
```

Dump Tags Example

> -----------------

Reviewer: ```suggestion
Database Dump Tags Example
--------------------------

Author: In my opinion the Database should not be known by a user
